plugin/helpers.go

```diff
@@ -10,6 +10,20 @@ type Helpers interface {
 	// the specifications of the passed bot.
 	// Returns the id of the bot created or existing.
 	EnsureBot(bot *model.Bot) (string, error)
+
+	// KVGetJSON retrievs a value based on the key.
+	KVGetJSON(key string, value interface{}) error
+
+	// KVSetJSON stores a key-value pair
+	KVSetJSON(key string, value interface{}) error
+
+	// VKCompareAndSetJSON updates a key-value pair if the current
```

Comment:
Typo:
```suggestion
	// KVCompareAndSetJSON updates a key-value pair if the current
```

Reply:
Hey @zkry -- looks like this typo is still there?

Reply:
I'll get this change in now. Thanks!